components/region_cache_memory_engine/src/background.rs

```diff
@@ -439,6 +439,7 @@ impl BackgroundRunnerCore {
     // if `false` is returned, the load is canceled
     fn on_snapshot_load_finished(&mut self, range: CacheRange) -> bool {
         fail::fail_point!("on_snapshot_load_finished");
+        fail::fail_point!("on_snapshot_load_finished2");
```

Comment:
What is the difference between these above two fail points?

Reply:
In the test in this PR, one is used to notify the execution is reached (by cfg_callback) and the other is used to pause the execution.

Reply:
How about using the bool flag as the trigger just like:
```
    fail_point!(
            "on_snapshot_load_finished_range",
            range != CacheRange(start_key, end_key),
            |_| {}
        );
```

Reply:
The notification and pause I mentioned above is for the same range.